RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Chromium.java

```diff
@@ -410,8 +424,14 @@ private void getCookie(String browser, String browserLocation) {
 
         FileManager fileManager = currentCase.getServices().getFileManager();
         List<AbstractFile> cookiesFiles;
+        String cookieFileName = COOKIE_FILE_NAME;
+        if (browser == NbBundle.getMessage(Chromium.class, "Browser.name.UC.Browser")) {
+            cookieFileName = "%" + COOKIE_FILE_NAME + "%";
+        }
         try {
-            cookiesFiles = fileManager.findFiles(dataSource, "%Cookies%", browserLocation); //NON-NLS
+            // Wildcard on front and back of Cookies as there are Cookie files that start with something else
+            // ie: UC browser has "Extension Cookies.9" as well as Cookies
```

Comment:
This comment seems strangely placed - at this point the cookieFileName may not have any % at all if it wasn't the UC browser

Reply:
Moved comment to better place